hypha/apply/projects/forms/vendor.py

```diff
@@ -0,0 +1,120 @@
+            except AttributeError:
+                pass
+            try:
+                self.fields[field].help_text = getattr(self.form_settings, f'{field}_help_text')
+            except AttributeError:
+                pass
+        return fields
+
+
+class CreateVendorFormStep1(BaseVendorForm, forms.Form):
+    TYPE_CHOICES = [
+        ('organization', 'Yes, the account belongs to the organisation above'),
+        ('personal', 'No, it is a personal bank account'),
+    ]
+
+    name = forms.CharField(required=True)
+    contractor_name = forms.CharField(required=True)
+    type = forms.ChoiceField(choices=TYPE_CHOICES, required=True, widget=forms.RadioSelect)
+
+    def __init__(self, *args, **kwargs):
+        super(CreateVendorFormStep1, self).__init__(*args, **kwargs)
+        self.fields = self.apply_form_settings(self.fields)
+
+
+class CreateVendorFormStep2(BaseVendorForm, forms.Form):
+    required_to_pay_taxes = forms.TypedChoiceField(
+        choices=((False, 'No'), (True, 'Yes')),
+        coerce=lambda x: x == 'True',
+        widget=forms.RadioSelect,
+        required=True
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(CreateVendorFormStep2, self).__init__(*args, **kwargs)
+        self.fields = self.apply_form_settings(self.fields)
+
+
+class CreateVendorFormStep3(FileFormMixin, BaseVendorForm, forms.Form):
+    due_diligence_documents = MultiFileField(required=True)
+
+    def __init__(self, *args, **kwargs):
+        super(CreateVendorFormStep3, self).__init__(*args, **kwargs)
+        self.fields = self.apply_form_settings(self.fields)
+
+
+class CreateVendorFormStep4(BaseVendorForm, forms.Form):
+    CURRENCY_CHOICES = [
+        (currency, f'{get_currency_name(currency)} - {currency}')
+        for currency in list_currencies()
+    ]
+
+    account_holder_name = forms.CharField(required=True)
+    account_routing_number = forms.CharField(required=True)
+    account_number = forms.CharField(required=True)
+    account_currency = forms.ChoiceField(
+        choices=CURRENCY_CHOICES,
+        required=True,
+        initial='USD'
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(CreateVendorFormStep4, self).__init__(*args, **kwargs)
+        self.fields = self.apply_form_settings(self.fields)
+
+
+class CreateVendorFormStep5(BaseVendorForm, forms.Form):
+    need_extra_info = forms.TypedChoiceField(
+        choices=((False, 'No'), (True, 'Yes')),
+        coerce=lambda x: x == 'True',
+        widget=forms.RadioSelect,
+        required=True
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(CreateVendorFormStep5, self).__init__(*args, **kwargs)
+        self.fields = self.apply_form_settings(self.fields)
+
+
+class CreateVendorFormStep6(BaseVendorForm, forms.Form):
+    CURRENCY_CHOICES = [
+        (currency, f'{get_currency_name(currency)} - {currency}')
+        for currency in list_currencies()
+    ]
```

Comment:
Can we get the list sorted A-Z and filtred to only include current currency. Looks like it now includes old obsolete currencies.